doc/tutorial.txt

```diff
@@ -155,7 +160,8 @@ used to create IO-bounded and memory-bounded busy loop. Default value is
 		"gnuplot" : false,
 		"io_device" : "/dev/null"
 		"mem_buffer_size" : 4194304,
-		"cumulative_slack" : false
+		"cumulative_slack" : false,
+		"forks_limit" : 1024
```

Comment:
Not sure what happened, but I could cause a segfault by slightly modifying example9 (notice how I recursively call thread3):
```
{
	/*
	 * A simple example on using fork event.
	 *
	 * thread3 will fork thread1 and thread2 in each of its phases.
	 * Note that thread2 will NOT be instantiated at start time and
	 * will only run when thread3 forks it. Unlike thread1 which will
	 * run at the start - since default value of instance property is
	 * 1 - and will run again when thread3 forks ie: twice in total.
	 */
	"tasks" : {
		"thread1" : {
			"phases" : {
				"phase1" : {
					"run" : 10000,
					"sleep" : 10000
				}
			}
		},
		"thread2" : {
			"instance": 0,
			"phases" : {
				"phase1" : {
					"run" : 20000,
					"sleep" : 20000
				}
			}
		},
		"thread3" : {
			"phases" : {
				"phase1" : {
					"fork": "thread1",
					"run" : 10000,
					"sleep" : 10000
				},
				"phase2" : {
					"fork": "thread3",
					"run" : 20000,
					"sleep" : 20000
				}
			},
			"loop": 10
		}
	},
	"global" : {
		"duration" : 60,
		"calibration" : "CPU0",
		"default_policy" : "SCHED_OTHER",
		"pi_enabled" : false,
		"lock_pages" : false,
		"logdir" : "/tmp/",
		"log_basename" : "rt-app1",
		"ftrace" : true,
		"gnuplot" : false
	}
}
```

Reply:
Hmm I can't reproduce this

Reply:
Have you applied/picked up the whole branch? I have fixed a segfault on SIGINT, you haven't hit that by any chance?

Reply:
Mmm, strange. Pretty sure I was on the last complete version, but I'm not able to reproduce it anymore. Fair enough. :-)

Reply:
If there's a monster lurking beneath it'll show its head eventually. But until it does, I'm afraid it'd be hard to do anything :-)